.github/autobump.txt

```diff
@@ -8,6 +8,7 @@ activemq
 ada-url
 adb-enhanced
 adios2
+advancemame:
```

Comment:
this seems wrong

Reply:
Didn't we remove `advancemame` due to the licensing issue?

Reply:
we did, shit. remove it now